clients/java/signalr/src/main/java/IHubProtocol.java

```diff
@@ -0,0 +1,11 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+public interface IHubProtocol {
+    String name();
+    int version();
+    TransferFormat transferFormat();
+    InvocationMessage parseMessage(String message);
```

Comment:
I think this needs to return an array.

Reply:
Yeah, if we pass the entire payload into the protocol than this should be an array